Resource Leak in SqlUtils.java

Cursor created at line 102 does not get closed.

libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/SqlUtils.java:112: error: RESOURCE_LEAK resource acquired to cursor by call to rawQuery(...) at line 102 is not released after line 112 (source | trace)

--- a/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/SqlUtils.java
+++ b/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/SqlUtils.java
@@ -118,6 +118,7 @@ public class SqlUtils {
             return true;
         } finally {
             db.endTransaction();
+            closeCursor(cursor);
         }
     }
 

